app/models/archived_patient.rb

```diff
@@ -144,3 +146,10 @@ def self.convert_patient(patient)
     archived_patient
   end
 end
+
+# check if a patient has been audited; allow them to be archived if they were NOT audited but were created more than two years ago
+def check_if_audited
+  unless self.audited
+    self.created_at > 2.years.ago ? true : false
```

Comment:
The date we want depends on how long a window they want for the auditing. @colinxfleming do you remember how long that was, or who we should ping to get it? My hunch is to check for the self.fulfillment date being six months ago.

Reply:
It's currently 120 days. Let's ask @alisonnjones or @verbingthenoun  how long that window should be in practice.